### ISSUE TITLE
ODF files preview

### ISSUE THREAD

**Issue by gericom-hummer** [meganz/webclient#24]
I'm using Mega to backup a lot of docs, I would love if Mega web client had a xls, doc, odt, ods, or in one word ODF preview mechanism, like in the Google Drive, Dropbox. below. It would make browsing and sharing docs will be much easier.

Thanks

**Comment by applesea2** [comment]
+1 for this. This is a dealbraker for many who don't want to install any software to write documents. It will also greatly open the window of businesses and users that can leverage office tools running in the browsers and collaborate with others

**Comment by eegh9Od2ahNg9aej** [comment]
Agreed. Especially, TXT! A bonus would be open source formats such as odt and ods.
